### ISSUE TITLE
alt_name, official_name és old_name figyelése az utcanevekhez

### ISSUE THREAD

**Issue by urbalazs** [vmiklos/osm-gimmisn#1660]
Az utcanevek általában a `name=*` kulcsban vannak megadva, azonban vannak további [névmegadási lehetőségek](https://wiki.openstreetmap.org/wiki/Key:name) is, mint például `alt_name=*`, `official_name=*` vagy `old_name=*`. Jó lenne, ha a hiányzó utcák és a további utcák funkció ezeket is nézné.

Használati eset:
Balatonszemesen a [Csonka Pál utca](https://www.openstreetmap.org/way/441928079) szépen ki van táblázva, még egy ismertető emléktábla is van az utca végén az utcanevet adó személyről. Ugyanakkor a Balatonszemesi Közös Önkormányzati Hivatal 1584/2012 ügyiratszámú dokumentumában ezt az utcát Kápolna köznek nevezték el, viszont ennek az információnak nyoma sincs a terepen, csak a referenciában szerepel. Ezért a Kápolna köz az `official_name=*` kulcsban lett megadva, viszont így ez a név szerepel a [hiányzó utcák](https://osm-gimmisn.vmiklos.hu/osm/missing-streets/balatonszemes/view-result) listájában.

**Comment by vasony** [comment]
A megoldás, hogy fel kell egy mappinget: 'Csonka Pál utca', 'Kápolna köz' és a probléma megoldva. 

Ha csak elnyomjuk a listákból ezeket, akkor lemaradunk az ott lévő házszámokról, tehát csak mappinggel lehet segíteni rajta. Tehát ha a referenciában Kápolna köz 1. van, és nem születik rá mapping, akkor nem fog megjelenni a hiányzó házszámok listáján.

A mappinget csinálhatná program "belül", de én automatán nem tenném, mert könnyen lehet, hogy egy alt_name vagy official_name ütközik valami mással, és nem biztos hogy teljes körűen lehetne ellenőrizni.
Először azt javasolnám, hogy ha akarunk ezzel foglalkozni, csak jelezzük az érintett felületeken, hogy fel kell rá venni egy mappinget, és döntsön a user.

**Comment by vmiklos** [comment]
Igen, nekem is a mapping jutott eszembe, ha ez nem egy sok helyen ismétlődő minta.  refstreets kulcs alatt tudsz ilyet felvenni.

**Comment by urbalazs** [comment]
Hozzáadtam három mappingot. [Így gondoltátok](https://github.com/vmiklos/osm-gimmisn/commit/2968e38fd84bb887fb935c8ebe514bd8a604351a)? Most az Arany János utca hiányzó, az Állomás tér meg a további utcáknál szerepel. Így csak rosszabb lett.

Az Arany János utca vége hivatalosan Nyár utca, de minden házszám az Arany János utcára szól a házszámtáblák szerint.
Az Állomás tér 1. és 3. validálható a vásárláskor kapott blokkok alapján, miközben a tér Baross Gábor térként van kitáblázva.

**Comment by vasony** [comment]
röviden: mapping akkor működik ha 1-1 ben meg lehet csinálni, tehát bármelyik oldal csak 1x van, itt nem ez a helyzet.

- Állomás tér: 1 közterületnek 1 neve lehet, arra kell tenni a házszámokat ami az utcán van, legyen az Állomás tér vagy Baross Gábor tér. Ellenkező az osm inspector is panaszkodni fog és nem fogod tudni a mappinget megcsinálni ha mind a 2 létezik osm-en, 
Ha mégis így hagyod, akkor egyszerűen csak ki kell szűrni az osm-street-filters részen a Baross Gábor teret.

- Arany János utca:
A referenciában szerepel az Arany János utca és ezen vannak a házszámok.
A referenciában szerepel a Nyár utca, múltkor vettem fel az önkormányzati határozat alapján

Több megoldás is van, 
- kiszedem a Nyár utcát a referenciából
- kiszedem az Arany János utcán a referenciából, és átrakom a házszámokat a Nyár utcára
- osm-street-filterrel kiszűrjük a Nyár utcát
